plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/segmented/RocksDBColumnarKeyValueStorage.java

```diff
@@ -186,15 +185,47 @@ public RocksDBColumnarKeyValueStorage(
    */
   private ColumnFamilyDescriptor createColumnDescriptor(
       final SegmentIdentifier segment, final RocksDBConfiguration configuration) {
-
+    boolean dynamicLevelBytes = true;
+    try {
+      ConfigOptions configOptions = new ConfigOptions();
+      DBOptions dbOptions = new DBOptions();
+      List<ColumnFamilyDescriptor> cfDescriptors = new ArrayList<>();
+
+      String latestOptionsFileName =
+          OptionsUtil.getLatestOptionsFileName(
+              configuration.getDatabaseDir().toString(), Env.getDefault());
+      LOG.trace("Latest OPTIONS file detected: " + latestOptionsFileName);
+
+      String optionsFilePath =
+          configuration.getDatabaseDir().toString() + "/" + latestOptionsFileName;
+      OptionsUtil.loadOptionsFromFile(configOptions, optionsFilePath, dbOptions, cfDescriptors);
```

Comment:
you cannot use loadLatestOptions ? directly instead of calling getLatestOptionsFileName and loadOptionsFromFile ?

Reply:
I think it is better for troubleshooting and check which file was used to load the options. You can have it if you enable trace logs.